JS/main_biz_owner.js

```diff
@@ -0,0 +1,91 @@
+import { initializeBizMap } from "./biz-map.js";
+import { initializeList } from "./biz-list_biz_owner.js";
+import { initFilters } from"./filter.js";
+import { initializeSearch } from "./search.js";
+
+const bizpointsResp = await fetch('data/union_sq.json');
+const bizpoints = await bizpointsResp.json();
+const map = initializeBizMap(bizpoints);
+
+initializeList(bizpoints);
+initFilters(bizpoints, map);
+initializeSearch(bizpoints);
+
+//dropdown
+const dropdown = L.control({ position: 'topright' });
+
+    dropdown.onAdd = function (map) {
+      const div = L.DomUtil.create('div', 'dropdown');
+  
+      div.innerHTML = '<button class="btn" id="filterButton">Filter</button>';
+      div.querySelector('#filterButton').addEventListener('click', toggleDropdown);
+      return div;
+    };
+  
+
+
+function toggleDropdown() {
+    const dropdownContent = document.getElementById("filterOptions");
+    dropdownContent.style.display = dropdownContent.style.display === "block" ? "none" : "block";
+  }
+
+  dropdown.addTo(map);
+
+// popup submission form
+
+function closeForm() {
+      document.getElementById("popup-form").style.display = "none";
+  }
+
+// Add this function to handle form submission
+function submitForm() {
+  // Get form data
+  const formData = {
+      name: document.getElementById("nameInput").value,
+      email: document.getElementById("emailInput").value,
+      phoneNumber: document.getElementById("phone-number").value,
+      message: document.getElementById("Message").value,
+      // Add more fields as needed
+  };
+
+  // Store form data in localStorage
+  localStorage.setItem("formData", JSON.stringify(formData));
+  console.log("Form submitted!");
+  // Close the form
+  closeForm();
+}
+

```

Comment:
First, you'll have to make the `submitForm` function available in the global scope by attaching it to the `window` object...
```suggestion
window.submitForm = submitForm;

```